tests/test_pypistats.py

```diff
@@ -747,3 +747,10 @@ def test_format_pandas(self) -> None:
         # Assert
         assert isinstance(output, pandas.DataFrame)
         assert str(output).strip() == expected_output.strip()
+
+    def test__package_not_exist(self) -> None:
+        package = "a" * 100
+        result = pypistats.python_major(package)
```

Comment:
Rather than making network calls and hitting the real API, please can you mock this, like the other tests?

Reply:
Sure!